docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,252 @@
+.. _make_hgrid_yaml:
+
+make_hgrid
+==========
+
+Structured YAML to run :ufs-utils:`make_hgrid<make-hgrid>` is validated by JSON Schema and requires the ``make_hgrid:`` block, described below. If ``make_hgrid`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML ``make_hgrid:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/make_hgrid.yaml
+
+UW YAML for the ``make_hgrid:`` Block
+-------------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+run_dir:
+^^^^^^^^
+
+The path to the directory where ``make_hgrid`` will write its outputs.
+
+config:
+^^^^^^^
+
+Describes the required parameters to run a ``make_hgrid`` configuration. 
+
+
+grid_type:
+""""""""""
+
+Specify type of topography. See the following options for more details:
+
+* from_file:              --my_grid_file must be specified.
+* spectral_grid:          no other optional or required arguments.
+* regular_lonlat_grid:    --nxbnds, --nybnds --xbnds, --ybnds, must be specified to define the grid bounds.
+* tripolar_grid:          --nxbnds, --nybnds, --xbnds, --ybnds, must be specified to define the grid bounds.
+* conformal_cubic_grid:   --nratio is optional argument.
+* gnomonic_ed:            equal distance gnomonic cubic grid.
+* simple_cartesian_grid:  --xbnds, --ybnds must be specified to define the grid bounds location and grid size. --simple_dx and --simple_dy must be specified to specify uniform cell length.
+* f_plane_grid:           For setting geometric fractors according f-plane. f_plane_latitude need to be specified
+* beta_plane_grid:        For setting geometric fractors according to beta plane. f_plane_latitude need to be specified 
```

Comment:
The white space in this section is not preserved when rendered. You might consider a table. Please also use the double backtick highlighting for any of the flags. The use of colons here also makes me think these should be entries in the YAML instead of punctuation.

Reply:
Thanks - the table format looks much better!